### DISCUSSION TITLE
Logging using OTEL Collector agent

### DISCUSSION THREAD

**Discussion by panthamm** [open-telemetry/opentelemetry-collector#3475] [Q&A]
Currently, we are in the process of integrating the OpenTelemetry to existing .Net applications in the Organization. We are able to achieve the Traces using OTEL collector agent. As per existing documentation, I don't see an OTEL collector agent is supporting the Logging. I would like to contribute to developing the logging functionality at the OTEL Collector Agent level. Please let me know how to proceed next.

**Comment by bogdandrutu** [comment]
@panthamm can you explain what exactly is missing?

**Comment by adriangb** [comment]
If I understand correctly, the question is "how can I emit logs that are OTEL compatible (and thus can be processed by the collector)"?
It is my understanding that the logging spec isn't finalized, and I'm not sure if it's integrated into the collector, but not 100% on that.

**Reply by panthamm** [reply]
Thanks for the information. I also learned same Log spec is not finalized. Right now we pursuing traces implementation using OpenTelemetry Standards.
